core/src/main/scala/cats/package.scala

```diff
@@ -147,6 +155,14 @@ package object cats {
   val Semigroup = cats.kernel.Semigroup
   val Monoid = cats.kernel.Monoid
   val Group = cats.kernel.Group
+  val Eq1 = cats.kernel.Eq1
+  val Eq2 = cats.kernel.Eq2
+  val Hash1 = cats.kernel.Hash1
+  val Hash2 = cats.kernel.Hash
```

Comment:
A typo maybe? Shouldn't it be `val Hash2 = cats.kernel.Hash2` instead?

Reply:
haha, yep! Thanks! I'll fix that up.

Reply:
fixed

Reply:
I guess the reason why it slipped through is that there were no tests for that ;)